src/renderer/channel/send-to-main.injectable.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import ipcRendererInjectable from "../app-paths/get-value-from-registered-channel/ipc-renderer/ipc-renderer.injectable";
+
+const sendToMainInjectable = getInjectable({
+  id: "send-to-main",
+
+  instantiate: (di) => {
+    const ipcRenderer = di.inject(ipcRendererInjectable);
+
+    return (channelId: string, message: any) => {
```

Comment:
messages can be arbitrary long list of arguments. Also, we have to handle mobx data types here too.

Reply:
1. At the moment we support one argument. It doesn't matter if IPC supports multiple. We don't. If you want to achieve same, it's better to use object instead.
2. MobX data-types are not welcome here. Typing should be "extends JSONSerializable`.

Reply:
Actually typings should be `JSONValue extends T ? T : never` but the problem with that is `Proxy` which is why we have the `toJS` function calls with the previous IPC implementation.

Reply:
Added requirement for messaging in channels to be serializable as `JsonValue`.